src/core/Akka.Tests.Performance/Actor/ActorThroughputSpecBase.cs

```diff
@@ -44,17 +44,15 @@ public void Benchmark(BenchmarkContext context)
                 _receiver.Tell(string.Empty);
                 ++i;
             }
-            _resetEvent.Wait(1000); //wait up to a second
+            _resetEvent.Wait(); //wait up to a second
```

Comment:
Used the same technique on the `ActorThroughputSpecBase` as I did on the `RemoteMessageThroughputSpecBase` to ensure more accurate readings.